### PR TITLE
Change to Nunito Sans

### PR DESCRIPTION
**Pull request by fsargent**
## Description
Updated typography to use a Sans Serif font.
Modified theme so that buttons could use lower case.

## Screenshots / Videos (frontend only) 
OLD:
image

NEW:
image

OLD:
image

NEW
image

### CODE REVIEW

File: packages/frontend/src/theme.tsx [hunk 1]
```diff
@@ -157,41 +157,19 @@ const brandPalette: PaletteOptions = {
...
-    fontFamily: 'Montserrat, Verdana, sans-serif',
-    marginTop: '1rem',
-    marginBottom: '1rem',
-  },
-  h6: {
-    fontFamily: 'Montserrat, Verdana, sans-serif',
-    marginTop: '1rem',
-    marginBottom: '1rem',
-  }
+  ...Object.fromEntries(['h1', 'h2', 'h3', 'h4', 'h5', 'h6'].map(level => [
```
Comment (ArendPeter) [inline packages/frontend/src/theme.tsx:165]: Nice! Much cleaner

File: packages/frontend/src/theme.tsx [hunk 2]
```diff
@@ -157,41 +157,19 @@ const brandPalette: PaletteOptions = {
...
+    level,
+    {
+      fontFamily: '"Nunito Sans", sans-serif',
```
Comment (ArendPeter) [inline packages/frontend/src/theme.tsx:168]: We chose Montserrat with Verdana as a fallback in order to be consistent with Equal Vote branding, those should also be I think those are also sans serif?

File: packages/frontend/src/theme.tsx [hunk 3]
```diff
@@ -157,41 +157,19 @@ const brandPalette: PaletteOptions = {
...
 
 const brandTypography: TypographyOptions = {
-  // fontFamily: 'Montserrat',
-  fontFamily: 'Verdana, sans-serif',
-  button:{
-    fontFamily: 'Montserrat, Verdana, sans-serif',
+  fontFamily: '"Nunito Sans", sans-serif',
+  button: {
+    fontFamily: '"Nunito Sans", sans-serif',
+    textTransform: 'none'
```
Comment (ArendPeter) [inline packages/frontend/src/theme.tsx:163]: I don't have a strong opinion about the all caps, I'm fine with removing the textTransform to see how it looks

**Review by ArendPeter — COMMENTED** [review]
Looks good!

Could you clarify the choice of Nunito sans? Monserrat and Verdana should also be sans-serif, and they have the benefit of mirroring the Equal Vote branding 

I'm happy to reconsider the font choice, but I wanted to have that context included

**Comment by fsargent** [comment]
Most of it is just that I think it looks better. I can remove the font choices and just keep the code improvements. The best thing is allowing the buttons to be lowercase, I think. I will say it doesn’t look like it matches equal.vote right now because of the color choices. Is that a design consideration?
Nunito sans may be a bit arbitrary. Suggestions welcome.

**Pull request merged**